jbdevstudio/com.jboss.jbds.central.discovery.earlyaccess/plugin.xml

```diff
@@ -65,6 +65,37 @@ appropriate tests + report issues to plug-in providers as required.</description
          <icon image32="images/arquillian_32.png"/>
          <overview url="http://www.jboss.org/tools"/>
       </connectorDescriptor>
-
+      
+      <!-- Windup -->
+      <connectorDescriptor
+            categoryId="org.jboss.tools.central.discovery.e.migration"
+            certificationId="com.jboss.jbds.discovery.certification.supported.earlyaccess"
```

Comment:
Here, you chose certification.supported.earlyaccess but for JBT you chose certification.experimental.earlyaccess ... was that intentional?